src/registrar/tests/test_views.py

```diff
@@ -378,3 +548,124 @@ def test_application_form_conditional_elections(self):
         election_page = type_result.follow()
         self.assertContains(election_page, TITLES["organization_election"])
         self.assertNotContains(election_page, TITLES["organization_federal"])
+
+    @skip("WIP")
+    def test_application_edit_restore(self):
+        """
+        Test that a previously saved application is available at the /edit endpoint.
+        """
+        ao, _ = Contact.objects.get_or_create(
+            first_name="Testy",
+            last_name="Tester",
+            title="Chief Tester",
+            email="testy@town.com",
+            phone="(555) 555 5555",
+        )
+        domain, _ = Domain.objects.get_or_create(name="city.gov")
+        alt, _ = Website.objects.get_or_create(website="city1.gov")
+        current, _ = Website.objects.get_or_create(website="city.com")
+        you, _ = Contact.objects.get_or_create(
+            first_name="Testy you",
+            last_name="Tester you",
+            title="Admin Tester",
+            email="testy-admin@town.com",
+            phone="(555) 555 5556",
+        )
+        other, _ = Contact.objects.get_or_create(
+            first_name="Testy2",
+            last_name="Tester2",
+            title="Another Tester",
+            email="testy2@town.com",
+            phone="(555) 555 5557",
+        )
+        application, _ = DomainApplication.objects.get_or_create(
+            organization_type="federal",
+            federal_type="executive",
+            purpose="Purpose of the site",
+            security_email="security@city.com",
+            anything_else="No",
+            is_policy_acknowledged=True,
+            organization_name="Testorg",
+            address_line1="address 1",
+            state_territory="NY",
+            zipcode="10002",
+            authorizing_official=ao,
+            requested_domain=domain,
+            submitter=you,
+            creator=self.user,
+        )
+        application.other_contacts.add(other)
+        application.current_websites.add(current)
+        application.alternative_domains.add(alt)
+
+        # prime the form by visiting /edit
+        url = reverse("edit-application", kwargs={"id": application.pk})
+        response = self.client.get(url)
+
+        url = reverse("application_step", kwargs={"step": "organization_type"})
+        response = self.client.get(url, follow=True)
+        self.assertContains(response, "<input>")
+        # choices = response.context['wizard']['form']['organization_type'].subwidgets
```

Comment:
**nitpick**: If we are going to comment out this much code, could we leave a TODO at the beginning of the block here about why it is commented out?

Reply:
Yup, can do. Also happy to finish these tests on this PR, if you'd like... but as you noted, the restore functionality is still somewhat tbd, so I would rather wait until the design for that is finalized.